webapp/src/routes/Home/DashboardSidebar.js

```diff
@@ -56,7 +56,7 @@ const PAGES = [
   },
   {
     title: 'news',
-    href: '/news',
+    href: 'https://medium.com/@MyvoteEOS',
```

Comment:
This link dosen't work on mobile

<img width="469" alt="image" src="https://user-images.githubusercontent.com/5632966/154808158-d1a57035-2879-4db6-88c2-e3fba1917e0f.png">